hack/defaults

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+TEMP_DIR=`mktemp -d`
+
+KUBEVIRT_MANIFEST_VERSION="v0.15.0"
+CDI_MANIFEST_VERSION="v1.6.0"
+
+function kubevirt_sed {
+    sed -i "s/          image: &image docker\.io\/kubevirt\/virt-operator:${KUBEVIRT_MANIFEST_VERSION}/          image: ${CONTAINER_REGISTRY}\/virt-operator:${KUBEVIRT_VERSION}/g" ${TEMP_DIR}/kubevirt-operator.yaml
```

Comment:
mutating the release manifests like this is going to cause an integration friction point. We'll break the assumptions being made here during a release and this will end up failing. 

we should be using some sort of templated manifest here and injecting variables into it. there's already a kubevirt-operator.yaml.j2 file exported for every kubevirt release.  That's an interface we can reasonably support other projects consuming.

Reply:
This script is meant to be a stopgap for folks that need some automation to deploy kubevirt and cdi operators.  A bunch of this code is going to get phased out quickly as the HCO is developed, so I was going for the "quick and dirty" approach.